android/src/main/java/com/twiliovoicereactnative/IncomingCallNotificationService.java

```diff
@@ -193,7 +197,6 @@ private void sendCallInviteToActivity(CallInvite callInvite, int notificationId)
   private void bringAppToForeground(String callSid, int notificationId, String uuid) {
     NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
     notificationManager.cancel(notificationId);
-    sendBroadcast(new Intent(Intent.ACTION_CLOSE_SYSTEM_DIALOGS));
```

Comment:
@mhuynh5757 - since we have removed this, I wonder how the app behavior will be when it receives a incoming notification when in the background. Can you please validate the behavior for AOS <12 and AOS≥ 12